api/services/YouTubeService.js

```diff
@@ -79,14 +83,27 @@ function search(query, maxResults) {
 }
 
 function nextRelated(key) {
+  return relatedVideos(key)
+    .then((videos) => videos[Math.floor(Math.random() * videos.length)].id.videoId);
+}
+
+function relatedVideos(key) {
   return new Promise((resolve, reject) => {
-    request(`https://www.googleapis.com/youtube/v3/search?relatedToVideoId=${key}&part=snippet&key=${process.env.GOOGLE_API_KEY}&maxResults=3&type=video`, (error, response, body) => {
+    request(`https://www.googleapis.com/youtube/v3/search?relatedToVideoId=${key}&part=snippet&key=${process.env.GOOGLE_API_KEY}&maxResults=10&type=video`, (error, response, body) => {
```

Comment:
I'll make the change but I was thinking that by changing it to 10 we'd probably stop running into the autoplay playing the same songs again and again